docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3"
+services:
+    app:
+        build:
+            context: .
+        ports:
+            - "8000:8000"
+        volumes:
+            - ./:/app
+        command: >
+            sh -c "python manage.py migrate &&
+                    python3 manage.py wait_for_db &&
+                      python manage.py runserver 0.0.0.0:8000"
```

Comment:
how will it migrate without db ?
it seems to me, that wait_for_db command should go before